jenkinsfiles/Jenkinsfile.rds-publishing

```diff
@@ -0,0 +1,17 @@
+elifePipeline {
+    stage 'Notify', {
+        elifeSlack "Publishing RDS articles on *prod* (<${env.RUN_DISPLAY_URL}|Build>)", "#deployments"
```

Comment:
Should be targeting `continuumtest`?

Reply:
It can be added; `preview` is our main environment so `prod` still need to be included to cater for it.

Reply:
Agreed not to add this, not to slow down the production team using this pipeline.